docs/source/config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "replacementPatterns": [
+        {
+            "pattern": ".md$",
+            "replacement": ""
+        },
+	{
+            "pattern": "$",
+            "replacement": ".md"
+	}
```

Comment:
What does this do?

Reply:
Nothing, file should be removed. Was part of the broken link checker.